interfaces/cython/cantera/composite.py

```diff
@@ -608,7 +623,7 @@ def append(self, state=None, **kwargs):
             raise IndexError("Can only append to 1D SolutionArray")
 
         for name, value in self._extra.items():
-            value.append(kwargs.pop(name))
+            np.append(value, kwargs.pop(name))
```

Comment:
This line still isn't fixed.

Reply:
I didn't think this through before,Anyways

> I think the simplest method is just to reassign `value`: `value = np.append(value, kwargs.pop(name))

Appending `np.ndarrays` is not as fast 

Turns out converting them to lists and appending does a better job!!, as you said before.

see https://stackoverflow.com/a/22394181 and https://stackoverflow.com/questions/7133885/fastest-way-to-grow-a-numpy-numeric-array

```
In [2]: %%timeit 
   ...: a = np.empty((0,3), int) 
   ...: l = list(a) 
   ...: for i in range(1000): 
   ...:     l.append([3*i+1,3*i+2,3*i+3]) 
   ...: l = np.array(l)                                                                                                                                                                                                   
467 µs ± 6.32 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

In [3]: %%timeit 
   ...: a = np.array([3,4,3]) 
   ...: for i in range(1000): 
   ...:     l = list(a) 
   ...:     l.append([3*i+1,3*i+2,3*i+3]) 
   ...:     l = np.array(l) 
   ...:                                                                                                                                                                                                    
2.27 ms ± 531 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)

In [4]: %%timeit 
   ...: a = np.empty((0,3), int) 
   ...: for i in range(1000): 
   ...:     a = np.append(a, 3*i+np.array([[1,2,3]]), 0) 
   ...:                                                                                                                                                                                                    
3.88 ms ± 21.2 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

```